server/db.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+require('dotenv').config();

```

Comment:
This is redundant if I'm not mistaken, require('dotenv').config() should only need to be performed once.